hassio/src/addon-view/config/hassio-addon-config-tab.ts

```diff
@@ -58,7 +59,11 @@ class HassioAddonConfigDashboard extends LitElement {
       haStyle,
       hassioStyle,
       css`
-        .content {
+        @media screen and (min-width: 1024px) {
+          .content {
+            width: 50%;
+            margin-left: 25%;
```

Comment:
You want it centered right?
```suggestion
            margin: auto;
```

Reply:
We sure do :)